ietf-layer0-types.yang

```diff
@@ -1533,11 +1533,18 @@ module ietf-layer0-types {
   grouping transceiver-mode {
     description
       "This grouping is intended to be used for reporting the
-       information of a transceiver's mode.
-       
-       The compatible-modes container shall be augmented with the 
-       proper leafrefs when used: see for example the 
-       transceiver-capabilities grouping below.";
+      information of a transceiver's mode.
+
+      The attributes for the explicit mode shall be augmented when
+      used with either:
+      - the proper leafrefs, when explicit mode templates are used;
+        or,
+      - the explicit-mode grouping, when explicit mode templates
```

Comment:
Checking with Esther, this case never happen: it will be always used explicit mode templates. So we should modify description.

Reply:
The description of the grouping is intended to allow designers to decide on a case-by-case basis when using this grouping whether to list the parameters here or via reference to a template.

For optical impairments the explicit mode is always described through templates but in other models (e.g., WDM interface model) it might be possible to add the parameters here with no need to define different groupings.